### DISCUSSION TITLE
Api for live chat

### DISCUSSION THREAD

**Discussion by kaflan** [jsforce/jsforce#1502] [Ideas]
Hello, @stomita tanks for this library. I have the question: chat in sales forces can integrate into API for developing?

**Comment by nylon22** [comment]
@stomita any traction on this? My team is also interested in an SDK for [Salesforce Live Agent Rest Api](https://help.salesforce.com/s/articleView?id=000340657&type=1) 

Would you be receptive to a pull request?

**Comment by paustint** [comment]
In the mean time, you can always use jsforce to make any arbitrary API request to Salesforce - you can read my details on this comment https://github.com/jsforce/jsforce/issues/922#issuecomment-546715026 

I attempted a PR to the docs for this, but it was never reviewed or merged - https://github.com/jsforce/jsforce-website/pull/10/files

**Comment by nylon22** [comment]
thanks!  @paustint
